exceptions.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, level, line_number, fixed_code, fixed_result):
                          level=level,
                          line_number=line_number,
                          fixed_code=fixed_code,
-                         fixed_result=fixed_result)
+                         fixed_result=fixed_result)  # what is the difference??
```

Comment:
Fixed code holds the fixed hedy code, f.e: `print hello world`. Fixed result holds the `ParseResult` object of parsing that code